README.md

```diff
@@ -8,3 +8,97 @@ Authors Haven - A Social platform for the creative at heart.
 ## Vision
 Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
+
+## Application Overview
+There is plenty of media in the world already. No matter what happens to “traditional media”, there’s nothing to stop the torrent of information rushing from smartphones, corporations, and media startups onto the Internet, available for the world to see.
+
+While it continues to be more and more efficient to put media-type stuff out there, we think there are big improvements to be made in a particular type of media “stuff”: That which is not necessarily personal and not necessarily news. That which we might just call ideas.
+
+What kind of ideas? Many kinds: A particular viewpoint on the happenings of the day (or of the past), hard-earned knowledge about how to do something better, a story that makes people laugh, smile, or feel something meaningful. If you have thoughts to share that you want to impact or influence people with—beyond just your friends and beyond 140 characters—we want to provide the tools and the place.
+
+## Application Features
+- [x] Users can see a landing page
+- [x] Users should be able to login to the application
+- [x] Users should be able to sign up to the application
+- [x] Users should be able to login and signup with social authentication
+- [x] Users should be able to reset a forgotten password
+- [x] Users should be able to create an article
+- [x] Users should be able to get a single article
+- [x] Users should be able to get and update their profile
+- [x] Users should be able to bookmark articles for later
+- [x] Users should be able to see a list of bookmarked articles
+- [x] Users should be able to see logistics of their articles
+- [x] Users can follow and unfollow an author
+- [x] Users can clap on article.
+- [x] Users can comment on an article.
+- [x] Users should be redirected to a 404 page if route does not exist
+- [x] Users can search and filter through the application
+
+## Project links
+- [Authors Haven repo](https://github.com/andela/apollo-ah-frontend.git)
+- [Pivotal-Tracker-Board](https://www.pivotaltracker.com/n/projects/2313263)
```

Comment:
I do not think you should make this public. If other developers would like to join to build this, then they can make contributions by either going through the listed issues and then raising a PR. Having them see this is not necessary. No need to expose the product roadmap this way.

Reply:
Okay, thank you for the feedback.